src/app/projects/project-details/project-details-build/project-details-build.component.ts

```diff
@@ -86,22 +88,30 @@ export class ProjectDetailsBuildComponent {
     }
   }
 
+  numberWithCommas(num: number) {
+    return this.numberFormat.format(num);
+  }
```

Comment:
Now that I see how this is used, suggest registering this as an Angular pipe instead, so the usage becomes something like this:

```html
      <td class="col-tests">
        {{build.testResultListSummary.failed | whNumberFormat}}<i class="pi pi-times"></i>&nbsp;
```

Look at our whDate pipe for reference

Reply:
Apparently you can use the built in `number` pipe like `{{ value | number: '1.0-0' }}`, so using that now.